system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.cpp

```diff
@@ -117,4 +118,36 @@ system_metrics_collector::ProcCpuData LinuxCpuMeasurementNode::makeSingleMeasure
   }
 }
 
+void LinuxCpuMeasurementNode::publishStatistics()
+{
+  if (publisher_ == nullptr) {
+    return;
+  }
```

Comment:
Is this a behavior we allow? If not, we should throw instead.

Reply:
Removed this check as this is a protected or private method.